src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -145,12 +145,11 @@ private function findAssetForBareImport(string $importedModule, AssetMapperInter
             return null;
         }
 
-        // remote entries have no MappedAsset
-        if ($importMapEntry->isRemotePackage()) {
-            return null;
+        if ($asset = $assetMapper->getAsset($importMapEntry->path)) {
+            return $asset;
         }
 
-        return $assetMapper->getAsset($importMapEntry->path);
+        return $assetMapper->getAssetFromSourcePath($importMapEntry->path);
```

Comment:
e.g. `bootstrap` imports `@popperjs/core`. If `bootstrap` is ultimately preloaded, this is what allows us to also preload `@popperjs/core`.